setting_up_the_database.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+engine = create_engine('sqlite:///films_db', echo=False)
```

Comment:
Minor comment:

It would be better to use a more explicit database file name with an extension, for example `sqlite:///films_db.sqlite3`. This makes the database configuration clearer and improves portability across environments.